src/components/snackbar/Snackbar.test.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { act, fireEvent, render, screen } from '@testing-library/react';
+
+import '@testing-library/jest-dom';
+
+import Snackbar from './Snackbar';
+
+describe('<Snackbar />', () => {
+  const props = {
+    open: true,
+    onclose: jest.fn(),
+    children: 'warning text',
+  };
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
+  });
+
+  it('Should have text', () => {
+    render(<Snackbar {...props} />);
```

Comment:
When rendering the elements with children, please wrap the parent like
```js
<Snackbar>This is a new text</Snackbar>
```

Reply:
Addressed the comment.